modules/veteran/app/controllers/veteran/v0/base_accredited_representatives_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module Veteran
   module V0
-    class AccreditedRepresentativesController < ApplicationController
+    class BaseAccreditedRepresentativesController < ApplicationController
```

Comment:
Removing the `verify_type` method from this class and leaving `before_action :verify_type` can be confusing at times. Usually there is a not implemented error added to the base class to endure sub classes implement. Or you can move the action to the sub controller classes.

Reply:
Thanks, Jon. Yeah, that was confusing. I moved the before_action call to the sub classes.